practice/src/js/products/productForm.js

```diff
@@ -75,41 +75,35 @@ export default class ProductForm {
 
       const product = formData.product;
 
-      try {
-        let response;
+      let response;
 
-        if (this.action === ACTIONS.ADD) {
-          response = await this.productService.add(product);
-        } else if (this.action === ACTIONS.EDIT) {
-          const productId = new URLSearchParams(window.location.search).get(
-            'id'
-          );
+      if (this.action === ACTIONS.ADD) {
```

Comment:
@JennyDoJD  saw that you have compared this.action === ACTIONS.ADD multiple times. Can we cache it?